manuscript/Cycle-Notation.md

```diff
@@ -29,7 +29,7 @@ leave out numbers that are not permuted. In our example we end up with the
 following cycle notation.
 
 ```plain
-(0 1 2)(3 4)
+(0 2 1)(3 4)
 ```
```

Comment:
The text states correctly that `0` goes to `1`, `1` goes to `2` and `2 goes back to `0`. That permutation has a cycle notation of `(0 1 2)`. The cycle `(0 2 1)` would send `0` to `2`, `2` to `1` and `1` back to `0`. It is in fact the inverse of the other permutation.

So together with the `(3 4)` would give the permutation `(0 1 2)(3 4)`.